### DISCUSSION TITLE
After Firmware Booting starts the ubuntu cloud image, there is no network in the virtual machine.

### DISCUSSION THREAD

**Discussion by lebron8dong** [cloud-hypervisor/cloud-hypervisor#6388] [Q&A]
./creat-cloud-init.sh
./target/release/cloud-hypervisor \
	--kernel ./hypervisor-fw \
	--disk path=focal-server-cloudimg-amd64.raw path=/tmp/ubuntu-cloudinit.img \
	--cpus boot=4 \
	--memory size=1024M \
	--net "tap=,mac=,ip=,mask="

**Comment by liuw** [comment]
You should see a virtio-net device in the guest, but you have to configure the interface yourself.
